### DISCUSSION TITLE
Why does it not support the Livp format?

### DISCUSSION THREAD

**Discussion by oldwang12** [immich-app/immich#8649] [Feature Request]
### The bug

Uploading photos in the format of livp through the iOS immich app will result in an error on the server.
```txt
[Nest] 1  - 04/08/2024, 1:22:53 PM   ERROR [AssetService] Unsupported file type IMG_4372.livp
```

I found that the code does not have the livp format here. Can you add the livp format here?

### The OS that Immich Server is running on

Ubuntu 22.10

### Version of Immich Server

v1.101.0

### Version of Immich Mobile App

v1.101.0

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
I am using Kubernetes, and the service information is consistent with the Docker Compose file you provided.
```

### Your .env content

```Shell
apiVersion: v1
kind: ConfigMap
metadata:
  name: immich-env
data:
  DB_PASSWORD: xxx
  DB_HOSTNAME: immich-postgres
  DB_USERNAME: root
  DB_DATABASE_NAME: immich
  REDIS_HOSTNAME: immich-redis
```

### Reproduction steps

```bash
1. upload picture xxx.livp will get err.
2.
3.
...
```

**Comment by oldwang12** [comment]
~~~

**Comment by W-Mai** [comment]
I've also encountered this problem with the latest version
